integrations/server/test_covidcast.py

```diff
@@ -188,10 +181,12 @@ def test_raw_json_format(self):
     row = self._insert_placeholder_set_one()
 
     # make the request
-    response, expected = self.request_based_on_row(row, **{'format':'json'})
+    response = self.request_based_on_row(row, **{'format':'json'})
+
+    expected = [row.as_dict(ignore_fields=CovidcastRow._api_row_compatibility_ignore_fields)]
```

Comment:
same as in `integrations/client/test_delphi_epidata.py`, this pattern (`.as_dict(ignore_fields=CovidcastRow._api_row_compatibility_ignore_fields)`) is ripe for shortcutting/reuse

Reply:
suggestion: another `as_api_compatibility_dict`